src/ios/SentryCordova.m

```diff
@@ -39,6 +39,25 @@ - (void)startWithOptions:(CDVInvokedUrlCommand *)command {
   [self.commandDelegate sendPluginResult:result callbackId:command.callbackId];
 }
 
+- (void)setEventOriginTag:(SentryEvent *)event {
+  if (event.sdk != nil) {
+    NSString *sdkName = event.sdk[@"name"];
+
+    // If the event is from cordova js, it gets set there and we do not handle it here.
+    if ([sdkName isEqualToString:cocoaSdk]) {
+      [self setEventEnvironmentTag:event origin:@"ios" environment:@"native"];
+    }
+  }
+}
+
+- (void)setEventEnvironmentTag:(SentryEvent *)event origin:(NSString *)origin environment:(NSString *)environment {
+  NSMutableDictionary *newTags = [NSMutableDictionary new];
+  [newTags addEntriesFromDictionary:event.tags];
```

Comment:
would this throw if `event.tags` is null?

Reply:
It shouldn't—but it does show a warning when opening the file in Xcode, let me add a nil check for cleanliness then